healthchain/pipeline/components/llm.py

```diff
@@ -0,0 +1,20 @@
+from healthchain.pipeline.components.base import Component
```

Comment:
Will this class still be relevant with the Spacy, HuggingFace etc. transformers?

Reply:
should they inherit from `Component`?